packages.dhall

```diff
@@ -1,14 +1,65 @@
 let upstream =
-      https://github.com/purescript/package-sets/releases/download/psc-0.13.8/packages.dhall sha256:0e95ec11604dc8afc1b129c4d405dcc17290ce56d7d0665a0ff15617e32bbf03
+      https://github.com/purescript/package-sets/releases/download/psc-0.15.4-20220725/packages.dhall
+        sha256:e56fbdf33a5afd2a610c81f8b940b413a638931edb41532164e641bb2a9ec29c
 
 let overrides = {=}
 
 let additions =
       { boxes =
-        { dependencies =
-          [ "profunctor", "prelude", "stringutils", "generics-rep", "strings" ]
+        { dependencies = [ "profunctor", "prelude", "stringutils", "strings" ]
         , repo = "https://github.com/cdepillabout/purescript-boxes.git"
-        , version = "v2.0.1"
+        , version = "v2.0.2"
```

Comment:
Should we use the commit hash for this one too?

Reply:
Tags seem a bit more stable to me